packages/core/cms/sections.json

```diff
@@ -233,18 +767,39 @@
             "properties": {
               "key": {
                 "title": "Key",
-                "description": "Tip: For collections, use: productClusterIds",
+                "hint": "For collections, use: productClusterIds",
```

Comment:
out of curiosity: where I can see all the available key types (like this `hint` option? I also didn't find this hint in the CMS (Admin).
<img width="737" alt="Screenshot 2023-04-04 at 18 33 09" src="https://user-images.githubusercontent.com/11325562/229927636-6f34610e-5aee-42fc-b19f-e54b31b1bef2.png">

Reply:
I changed to description, thanks!

![Screenshot 2023-04-05 at 12 17 46](https://user-images.githubusercontent.com/51174217/230126321-190a07fa-e64d-497a-ab8b-e1dc6b76842d.png)